BZAccessControl/hardhat.config.js

```diff
@@ -0,0 +1,39 @@
+require("dotenv").config();
+require("@nomicfoundation/hardhat-toolbox");
+require("@nomiclabs/hardhat-web3");
+require("hardhat-gas-reporter");
+
+/** @type import('hardhat/config').HardhatUserConfig */
+module.exports = {
+  solidity: {
+    version: "0.8.9",
+    settings: {
+      optimizer: {
+        enabled: true,
+        runs: 100,
```

Comment:
For modules, lets use default runs and compile version will be always latest => 0.8.17

Reply:
default runs = 200

Reply:
Updated hardhat configuration